index.js

```diff
@@ -124,6 +124,8 @@ if (dryRun) {
 const cutoffDate = moment().subtract(numDaysOld, 'd').format('YYYY-MM-DD');
 console.log(`Getting issues that haven't been touched since ${cutoffDate}`);
 
+var pickedIssues;
```

Comment:
@estellecomment OK then use `let`, that is `var` but with correct scoping rules. There should be almost no need to ever use `var` anymore :-)